app/modules/tmr.c

```diff
@@ -151,6 +151,7 @@ static timer_t tmr_get( lua_State *L, int stack ) {
 	if (lua_isuserdata(L, stack)) {
 		return (timer_t)luaL_checkudata(L, stack, "tmr.timer");
 	} else {
+		platform_print_deprecation_note("Use the dynamic timer API rather than the static one.", "end of 2016");
```

Comment:
Maybe it will be helpful to print exact method (or at least module) that generates warning, for users who migrates large existing project with lots of code.